### PR TITLE
Logs-collector new filelog collector and ingress grok parser

### PR DESCRIPTION
**Pull request by Kuckkuck**
* new 0.107.0 release build
* Switching from old filelog container collector to new auto discovery file format collector
* Adding grok parser for nginx ingress logs based on label 

`resource.attributes.app.label.name" == "ingress-nginx"`

### CODE REVIEW

File: Dockerfile.otel-collector [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:latest as journal
+FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:12.6 AS journal
 RUN apt update  \
   && apt upgrade -y  \
   && apt autoremove -y \
   && apt install -y systemd libssl-dev
-FROM --platform=${BUILDPLATFORM:-linux/amd64} ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:latest
+FROM --platform=${BUILDPLATFORM:-linux/amd64} ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:0.107.0
```
Comment (viennaa) [inline Dockerfile.otel-collector:6]: Why did you do the switch from latest to a static version?

**Comment by viennaa** [comment]
Resolve the conflicts please.

File: Dockerfile.otel-collector (hunk 1 above)
Comment (Kuckkuck) [reply Dockerfile.otel-collector:6]: To know on which version we are running on.

**Review by viennaa — APPROVED** [review]
more processors!11

**Pull request merged**